### DISCUSSION TITLE
PR compile failed!

### DISCUSSION THREAD

**Discussion by niuzhi** [open-policy-agent/community#325] [Community]
Hello.My PR failed to compile and reported the following error, how to solve it?

![image](https://user-images.githubusercontent.com/33443724/213338619-c78a65f9-cd8a-47c4-9505-883a69df586a.png)

**Comment by niuzhi** [comment]
PR：https://github.com/open-policy-agent/opa/pull/5570

**Comment by charlieegan3** [comment]
Hi @niuzhi, it looks like there was a failure in Github Actions during that run (there might not be any thing wrong in the PR). We can reopen and chat about this on your PR 🙂
